src/sentry/web/frontend/create_organization_member.py

```diff
@@ -16,20 +17,34 @@
 class CreateOrganizationMemberView(OrganizationView):
     required_scope = 'org:write'
 
-    def get_form(self, request, organization):
+    def get_form(self, request, organization, all_teams, allowed_roles):
         initial = {
             'role': organization.default_role,
+            'teams': [],
         }
 
         if settings.SENTRY_ENABLE_INVITES:
             form_cls = InviteOrganizationMemberForm
         else:
             form_cls = AddOrganizationMemberForm
```

Comment:
I need to verify this works with `AddOrganizationMemberForm` (probably not) – this has only been tested with `InviteOrganizationMemberForm`.